src/post/post.jl

```diff
@@ -0,0 +1,156 @@
+"""
+    e4st_post(post_config)
+
+Runs post-processing on `post_config`.  See [`read_post_config`](@ref) for information about the `post_config`.  The general outline of post processing is:
```

Comment:
doesn't look there is much for doc strings for read_post_config, did you mean to include more there?